src/Features/Core/Portable/DocumentHighlighting/AbstractDocumentHighlightsService.cs

```diff
@@ -157,13 +159,20 @@ private static bool ShouldConsiderSymbol(ISymbol symbol)
 
             var additionalReferences = new List<Location>();
 
-            foreach (var document in documentsToSearch)
+            foreach (var additionalDocument in documentsToSearch)
```

Comment:
I don't like the name additionalDocument just because we already have that term meaning something else entirely different (analyzer additional documents.)